src/d2_docker/config/dhis2-core-start.sh

```diff
@@ -54,10 +56,26 @@ run_sql_files() {
     find "$base_db_path" -type f \( -name '*.sql' \) |
         sort | while read -r path; do
         echo "Load SQL: $path"
-        $psql_cmd <"$path" || true
+        run_psql_cmd "$path"
+        exit_code=$?
```

Comment:
this won't work for set -e scripts. Safer:

```
exit_code=0
run_psql_cmd "$path" || exit_code=$?
```

Reply:
Done and tested